src/assets/scss/index.scss

```diff
@@ -0,0 +1,24 @@
+// Settings - used with preprocessors and contain font, colors definitions, etc.
+@import './settings/settings.variables';
+
+// Tools – globally used mixins and functions. It’s important not to output any CSS in the first 2 layers.
+
+// Generic – reset and/or normalize styles, box-sizing definition, etc. This is the first layer which generates actual CSS.
+@import './generic/generic.animation';
+
+// Elements – styling for bare HTML elements (like H1, A, etc.). These come with default styling from the browser so we can redefine them here.
+
+// Objects – class-based selectors which define undecorated design patterns, for example media object known from OOCSS
+
+// Components – specific UI components. This is where the majority of our work takes place and our UI components are often composed of Objects and Components
+@import './components/components.modeler';
+@import './components/components.toolbar';
+@import './components/components.inspector';
+@import './components/components.controls';
+@import './components/components.sequenceFlowFormSelect';
+@import './components/components.statusBar';
+@import './components/components.miniPaper';
+@import './components/components.validationStatus';
```

Comment:
Is there any way of importing all files matching a name pattern or in a directory? The risk here is that people will forget to add new SCSS files to this list which could be a pain point in the future.